src/components/widget/widget.directive.coffee

```diff
@@ -75,16 +75,22 @@ module.directive('impacWidget', ($templateCache, ImpacUtilities, ImpacWidgetsSvc
     },
     controller: 'ImpacWidgetCtrl',
     link: (scope, element) ->
-
       # initialize scope attributes
       # --------------------------------------
-      scope.widget.isLoading = true
-      scope.widget.settings = []
-      scope.pdfMode = false
-      scope.cssClass = ImpacUtilities.fetchWidgetCssClass(scope.widget)
-      scope.templatePath = ImpacUtilities.fetchWidgetTemplatePath(scope.widget)
+      onInit = ->
+        scope.showInfoPanel = false
+        scope.pdfMode = false
+        scope.widget.isLoading = true
+        scope.widget.settings = []
+        # Unused so far -->
+        scope.widget.hasEditAbility = true
```

Comment:
I think this is old code that has been removed with the implementation of ACLs (v1.7.0) => can you remove plz?

Reply:
Sure 👍